src/main/java/me/tomsdevsn/hetznercloud/objects/general/Subnet.java

```diff
@@ -1,15 +1,21 @@
 package me.tomsdevsn.hetznercloud.objects.general;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Please explain why you would add these three annotations. I can't see the reason to add them.